data/src/main/java/com/msg/data/remote/datasource/self_study/SelfStudyDataSourceImpl.kt

```diff
@@ -0,0 +1,23 @@
+package com.msg.data.remote.datasource.self_study
+
+import com.msg.data.remote.network.SelfStudyApi
+import com.msg.data.remote.util.safeApiCall
+import javax.inject.Inject
+
+class SelfStudyDataSourceImpl @Inject constructor(
+    private val selfStudyApi: SelfStudyApi
+): SelfStudyDataSource {
+    override suspend fun banSelfStudy(role: String, userId: Long) = safeApiCall {
+        selfStudyApi.banSelfStudy(
+            role =  role,
+            userId =  userId
```

Comment:
```suggestion
        selfStudyApi.banSelfStudy(
            role = role,
            userId = userId
```
뭐야 공백 부분들이 왜케 뭔가가 이상해요